src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -17,6 +17,36 @@
           }
         ]
       },
+      "logicalLocations": {
+        "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()": [
+          {
+            "name": "app",
+            "kind": "android-module"
```

Comment:
where'd you produce this name? i thought an android module referred to a discrete unit of functionality, so I'm surprised to see it as a container. does package below refer to a java package? i know you didn't contrive these names, but such is life when you touch a converter. are you proposing to add android-module to our like of kinds? would 'module' be sufficient (already defined)?